include/magma/app/Option.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file Option.hpp
```

Comment:
Why use one hpp file? Do you think we will don't expand functionality?

Reply:
It's a file for a template class